sync-core/src/test/java/com/cloudant/sync/replication/AttachmentsPushTest.java

```diff
@@ -216,6 +216,48 @@ public void pushAttachmentsTest2() throws Exception {
 
     }
 
+    @Test
```

Comment:
Might be worth mentioning the bug number and the Cloudant Sync groups URL that describe the problem this test is checking for?